migration/migrate_provider_from_study.sql

```diff
@@ -0,0 +1,137 @@
+        end if;
+        remote_databases := false;
+    elsif exists(SELECT datname FROM pg_catalog.pg_database WHERE datistemplate = false and datname = fn.study_name) then
+        raise notice 'separate databases structure detected';
+        if current_database() != study_name then
+            raise exception 'Invalid current database "%"', current_database() using hint='Assuming script launch at "'||study_name||'.*.*"', errcode='invalid_database_definition';
+        end if;
+        remote_databases := true;
+        create extension if not exists postgres_fdw;
+    else
+        raise exception 'Can''t detect type of database' using
+            hint='Is it the good database?',
+            errcode='invalid_database_definition',
+            detail='Can''t find schema nor database "study" from current session, use to determine the database structure.';
+    end if;
+
+    foreach params in array migrate loop
+        raise debug 'migration data = %', params;
+        /*Note: quote_indent() ⇔ %I ≠ quote_literal() ⇔ %L */
+        begin
+
+            if remote_databases then
+                execute format('create server if not exists %s foreign data wrapper postgres_fdw options (dbname %L)', concat('lnk_', params->>'to_schema'), params->>'to_schema');
+                execute format('create user mapping if not exists for %s server %s options (user %L)', current_user, concat('lnk_', params->>'to_schema'), current_user);
+            end if;
+
+
+            raise debug 'debug table input = %', params;
+
+            if remote_databases then
+                -- rename for potential conflict with foreign table name
+                execute format('alter table if exists %I rename to %I', params->>'to_table', concat(params->>'to_table', '_old'));
+                execute format('import foreign schema %I limit to (%I) from server %s into %I', 'public', params->>'to_table', concat('lnk_', params->>'to_schema'), 'public');
+                path_from := concat(quote_ident('public'), '.', quote_ident(concat(params->>'from_table')));
+                path_to := concat('public.', quote_ident(params->>'to_table'));
+            else
+                path_from := concat(study_name, '.', quote_ident(params->>'from_table'));
+                path_to := concat(quote_ident(params->>'to_schema'), '.', quote_ident(params->>'to_table'));
+            end if;
+            raise debug 'table locations: study=% src="%" dst="%"', study_name, path_from, path_to;
+            raise notice 'copy data from % to %', path_from, path_to;
+            execute format('update %s set %s = %s from %s where public.study.load_flow_parameters_uuid = public.load_flow_parameters.id',
+							path_to, params->>'to_column', concat(path_from, '.', params->>'from_column'), path_from, concat(path_from, '.', params->>'src_id_column'), concat(path_to, '.', 'dest_id_column')); --... on conflict do nothing/update
```

Comment:
wrong request+format
```suggestion
            execute format('update %s set %I = %s from %s where %s.%I = %s.%I',
                           path_to, params->>'to_column', concat(path_from, '.', quote_ident(params->>'from_column')), path_from, concat(path_from, '.', quote_ident(params->>'src_id_column')), concat(path_to, '.', quote_ident(params->>'dest_id_column')));
```

You also have tabs instead of space.